app/src/main/res/layout/profile_progress_header.xml

```diff
@@ -47,6 +47,7 @@
       android:layout_height="wrap_content"
       android:layout_marginTop="12dp"
       android:fontFamily="sans-serif-medium"
+      android:gravity="center"
       android:text="@{viewModel.profile.name}"
```

Comment:
Is it already centered for other landscape and tablet layouts?
Please also add the screenshots for landscape and tablet layouts.

Reply:
as discussed with @rt4914  earlier in the issue, the profile need not be centre aligned in other layouts i.e `layout-land`, `layout-sw600dp-port` and `layout-sw600dp-land`. So, I did not change them.

Reply:
Alright then, you can resolve this conservation.

Reply:
okay. thanks.